xknx/devices/state_updater.py

```diff
@@ -0,0 +1,52 @@
+"""Module for keeping the value of a RemoteValue from KNX bus up to date."""
+import asyncio
+
+
+class StateUpdater():
+    """Class for reading the value of a RemoteValue from KNX bus."""
+
+    def __init__(self,
+                 xknx,
+                 expiration_min=60,
```

Comment:
maybe just call it timeout?

Reply:
or update_interval?

Reply:
👍 I like update_interval! Very descriptive.

Reply:
`update_interval` 👍